app/home/controllers.js

```diff
@@ -12,6 +13,18 @@ async function dashboard(req, res) {
     whatsNewContent = null
   }
 
+  if (req.filterOutgoing[0].value === 0 || req.filterOutgoing[0].value >= 2) {
```

Comment:
What does this actually mean i.e.  Can it be given a meaningful name? 

`req.filterOutgoing[0].value === 0 || req.filterOutgoing[0].value >= 2`